core/src/banking_stage/leader_slot_timing_metrics.rs

```diff
@@ -5,7 +5,6 @@ use {
 
 #[derive(Default, Debug)]
 pub struct LeaderExecuteAndCommitTimings {
-    pub collect_balances_us: u64,
```

Comment:
we lose this metric, but I think it's fine - no realistic leader is running with this option enabled.

Reply:
but that's not reported directly by the workers - so we've lost the metric on the threads that execute when we're leader.